core/unsafe/console.js

```diff
@@ -33,16 +37,46 @@ colours.setTheme({
     title: ['green', 'bold']
 });
 
+var getTimestampString = function() {
+	var dt = new Date(),
+		diff = (dt.getTime() / 1000) - startupTime,
+		time = '[' + ('          ' + diff.toFixed(2)).slice(-10) + '] ';
```

Comment:
Actually not really. No matter what, using the built in classes we are going to get a diff in milliseconds (Js DateTime - Js DateTime = int64), by storing the value that won't change as an integer we avoid a repetitive internal DateTime to integer operation. Additionally, as this value is timezone agnostic, unlike using classes it will never run into timezone issues.